code/__HELPERS/mobs.dm

```diff
@@ -114,6 +116,14 @@
 		if(i != attempts_to_find_unique_name && !findname(.))
 			break
 
+/proc/random_unique_moth_name(gender, attempts_to_find_unique_name=10)

```

Comment:
remove the gender argument from this proc

Reply:
alsowhy